README.md

```diff
@@ -101,22 +108,253 @@ serial port name.
 
 ```js
 var ourBoard = new require('openbci-sdk').OpenBCIBoard();
-ourBoard.autoFindOpenBCIBoard(function(portName,ports) {
-    if(portName) {
+ourBoard.autoFindOpenBCIBoard().then((value) => {
+    if(Array.isArray(value)) {
```

Comment:
yea so you read it correctly! If there are no ports i would presume that you would get an empty array. Good question! Could you elaborate on your suggestion? Thanks!